website/docs/build-apps/how-to-guides/create-drill-down-view.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Create a detailed view using a modal or a new page.
+---
+
+# Create Data Drill down View
+
+This page shows you how to build a data drill down view from a master dataset.
+
+## Prerequisites
+- A datasource containing the data to display.
+- A query to fetch data from the datasource.
+
+## Create drill down view using list
+
+<div style={{ position: "relative", paddingBottom: "calc(50.520833333333336% + 41px)", height: "0", width: "100%" }}>
+  <iframe src="https://demo.arcade.software/d0YRgvug3zTUyniWTHKj?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{ position: "absolute", top: "0", left: "0", width: "100%", height: "100%", colorScheme: "light" }} title="Appsmith | Connect Data">
+  </iframe>
+</div>
+
+To create a data drill down view using a List widget, follow these steps:
+1. Drag and drop a List widget on to the canvas.
+2. Set the **Items** of the widget under **Data** in the property pane, using the following code where `fetch_passengers` is the fetch query:
+   ```jsx
+   {{fetch_passengers.data}}
+   ```
+3. Drag and drop a Container widget on to the canvas and disable the widget's **Visible** property in the property pane.
```

Comment:
I think we can combine steps 3 and 4. adding required widgets does not have to specified. Also we can use the examples for that usecase,  let's not talk about the usecase in the steps. eg: Passenger details.